common/helpers.hpp

```diff
@@ -129,15 +132,32 @@ namespace helpers
         std::random_device                 rd{};
         std::mt19937                       gen{rd()};
         std::uniform_int_distribution<int> distrib{-range, range};
-        distrib(gen); // prime generator to remove warning
+        (void)distrib(gen); // prime generator to remove warning
```

Comment:
Just curious, what warning does this line resolve?

Reply:
You mean the change (void) for unused result, or the prime generator?   std::uniform_int_distribution is same as rocblas usage would have to look at larger context.